src/main/scala/pipelines/text/AmazonReviewsPipeline.scala

```diff
@@ -71,11 +72,11 @@ object AmazonReviewsPipeline extends Logging {
     val conf = new SparkConf().setAppName(appName)
     conf.setIfMissing("spark.master", "local[2]") // This is a fallback if things aren't set via spark submit.
 
-    val sc = new SparkContext(conf)
+    val spark = SparkSession.builder.config(conf).getOrCreate()
```

Comment:
If we are doing this, can we file a new issue to update all examples to use SparkSession ? Kind of weird to only do it in one place etc.